src/ansys/pre_commit_hooks/add_license_headers.py

```diff
@@ -355,48 +377,30 @@ def add_hook_changes(before_hook: str, after_hook: str) -> None:
     after_hook: str
         Path to file after add-license-headers was run.
     """
-    # Compare each line of the file before and after add-license-headers was run,
-    # and combine the files into one
-    with open(before_hook, "r") as before_edit, open(
-        after_hook, "r"
-    ) as after_edit, NamedTemporaryFile(mode="w", delete=False) as tmp:
-        for before_edit_line, after_edit_line2 in zip(before_edit, after_edit):
-            # If the lines are different,
-            if before_edit_line != after_edit_line2:
-                # If the after_edit_line is empty, and the before_edit_line is the
-                # same as the next after_edit_line, remove the space by writing the
-                # before_edit_line. For example:
-                #
-                # before_edit file                 after_edit file
-                # line 1: """                      line 1:
-                # line 2: print("example")         line 2: """
-                # We want to remove the blank space, so we take line 1 from the before_edit file.
-                if not after_edit_line2.strip() and (before_edit_line == next(after_edit)):
-                    tmp.write(before_edit_line)
-                # If the before_edit_line is empty, write that line and the next line in
-                # the before_edit file. For example:
-                #
-                # before_edit file                 after_edit file
-                # line 1:                          line 1: print("example")
-                # line 2: print("example")         line 2: print("example 2")
-                # We want to keep the blank space, so we take lines 1 & 2 from the before_edit file.
-                elif not before_edit_line.strip():
-                    tmp.write(f"{before_edit_line}{next(before_edit)}")
-                # Otherwise, write the line from the after_edit file
-                else:
-                    tmp.write(after_edit_line2)
+    count = 0
+    before_hook_file = open(before_hook, "r", encoding="utf8")
+    before_hook_lines = before_hook_file.readlines()
+    found_reuse_info = False
+
+    # Copy file content before add-license-header was run into
+    # the file after add-license-header was run.
+    for line in fileinput.input(after_hook, inplace=True, encoding="utf8"):
+        # Copy the new reuse lines into the file
+        if _util.contains_reuse_info(line):
+            count += 1
+            found_reuse_info = True
+            print(line.rstrip())
```

Comment:
This print should be removed... right?

Reply:
Same with the other print statements?

Reply:
With fileinput inline replacement, the print statements are printed into the file that's being edited instead of to the console